presto-docs/src/main/sphinx/release/release-0.99.rst

```diff
@@ -0,0 +1,11 @@
+============
+Release 0.99
+============
+
+General Changes
+---------------
+* Optimize locking in TaskExecutor.
+* Fix reading maps with null keys from ORC.
+* Fix precomputed hash optimization for nulls values.
+* Make ``contains()`` parametric.
+* Revert to an older version of Jetty.
```

Comment:
I'd leave this one out, since I don't think we documented the upgrade to the new version of Jetty.